wordpress-mobile/WordPress-Android
# Experiment - Gutenberg Android X migration

Updates the Gutenberg mobile ref to point to the latest work, including the effort to migrate to AndroidX.

Gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1112

To test:
* Build and run WPAndroid and check that the block editor is working as normal.

Update release notes:
